build.gradle.kts

```diff
@@ -150,6 +151,13 @@ dependencies {
     testImplementation("com.nimbusds:nimbus-jose-jwt:$nimbusdsVersion")
     testImplementation("com.opentable.components:otj-pg-embedded:$postgresEmbeddedVersion")
     testImplementation("io.mockk:mockk:$mockkVersion")
+
```

Comment:
Skal appen bruke både junit og spek? Hadde kanskje vært greit å velge en av dem? :)

Reply:
Ja, det er fordi jeg har stjelt noen av (dine?) tester for PDL fra egenmeldt, og jeg hadde ikke så lyst til å skrive dem om...

Reply:
Du kan kanskje beholde logikken, men skrive om til junit..? Spek er jo bare innpakkingen uansett :)